test/regress/expect/ce/ce0.out

```diff
@@ -0,0 +1,4 @@
+Total cost: 6005
```

Comment:
is it possible to merge ce0.out, ce0.out ... -> ce.out? Otherwise, we have too many files.